packages/fakta-beregning/src/BeregningFaktaIndex.tsx

```diff
@@ -64,12 +86,26 @@ const BeregningFaktaIndex = ({
   submittable,
   erOverstyrer,
   arbeidsgiverOpplysningerPerId,
-}) => {
+  beregningErBehandlet,
+}: OwnProps) => {
   const skalBrukeTabs = beregningsgrunnlag.length > 1;
   const [aktivtBeregningsgrunnlagIndeks, setAktivtBeregningsgrunnlagIndeks] = useState(0);
   const aktivtBeregningsgrunnlag = beregningsgrunnlag[aktivtBeregningsgrunnlagIndeks];
+  const beregningsgrunnlagVilkår = vilkar.find(
+    vilkår => vilkår?.vilkarType?.kode === vilkarType.BEREGNINGSGRUNNLAGVILKARET,
+  );
+
+  if (beregningErBehandlet === false) {
```

Comment:
er typen `false` viktig her eller holder det med `! beregningErBehandlet`?

Reply:
ja, for denne komponenten brukes også i frisinn og omsorgspenger, og siden hooken ikke settes til true eller false i de sakene så ville beregning aldri blitt vist